application/config/migration.php

```diff
@@ -1,6 +1,6 @@
 <?php defined('BASEPATH') OR exit('No direct script access allowed');
 
 $config['migration_enabled'] = TRUE;
-$config['migration_type'] = 'sequential';
-$config['migration_version'] = 0;
+$config['migration_type'] = 'timestamp';
```

Comment:
I'm surprised the migration worked in CI, I didn't think we could switch from `sequential` to `timestamp` once we had picked one.

In any case, I'd prefer to we stick to `sequential`, if that's what we started with. I don't care about specifically _which_ it is, I care about it being consistently the same.

Reply:
In that case, yeah, let's leave it `sequential`, and ask @peterjdann to update his PR to use `sequential` as well.